common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/DiagnosisKey.java

```diff
@@ -236,4 +236,20 @@ public int hashCode() {
     result = 31 * result + Arrays.hashCode(keyData);
     return result;
   }
+
+  @Override
+  public String toString() {
+    return "DiagnosisKey{"
+        + "keyData=HIDDEN"
+        + ", rollingStartIntervalNumber=" + rollingStartIntervalNumber
+        + ", rollingPeriod=" + rollingPeriod
+        + ", transmissionRiskLevel=" + transmissionRiskLevel
+        + ", submissionTimestamp=" + submissionTimestamp
+        + ", consentToFederation=" + consentToFederation
+        + ", originCountry='" + originCountry + '\''
```

Comment:
if you need to add ' and use string+ anyhow, why don't use "'" then? reading escape sequences, is often a hard task

Reply:
Thanks for the hint, it's fixed